tox.ini

```diff
@@ -165,14 +165,12 @@ deps = gitpython >= 2.1.10
        oauth2client >= 4.1.2
 commands = python {toxinidir}/tasks/notify.py
 
-[testenv:dev]
-description = generate a DEV environment
+[testenv:devenv]
```

Comment:
I would be more inclined to update the documentation and make it dev instead.

Reply:
I prefer devenv.

Reply:
I personally believe the name `devenv` is a bit redundant and non-standard in this case. No other environments are suffixed with `env` and it seems to me to be a little pedantic. I'd vouch for using `dev` in this case, unless there is any other reason besides personal preference.

Reply:
It's personal preferenve against personal preference here, I guess :)